docs/fastddscli/cli/cli.rst

```diff
@@ -74,7 +75,7 @@ Where the optional parameters are:
 +--------------------------+-------------------------------------------------------------------------------------------+
 | ``-p  --port``           | UDP port chosen to listen the clients. Defaults to '11811'.                               |
 +--------------------------+-------------------------------------------------------------------------------------------+
-| ``-b  --backup``         | Creates a BACKUP *server*                                                                 |
+| ``-b  --backup``         | Creates a BACKUP *server* (see :ref:`discovery_general_settings`)                         |
```

Comment:
```suggestion
| ``-b  --backup``         | Creates a BACKUP *server* (see :ref:`discovery_protocol`)                         |
```

It might be clearer linking to the next subsection.